### PR TITLE
GB5 change password

### PR DESCRIPTION
**Pull request by kris-kolosek**

### CODE REVIEW

File: app/views/profiles/edit.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class='edit-profile-content'>
   <%= form_for(@profile, :html => {:multipart => true}) do |f| %>
-    <header class="layout-header">
+    <header class="layout-header-profile">
```
Comment (salac-kolosek) [inline app/views/profiles/edit.html.erb]: Class names should contain 2 parts.

Comment (kris-kolosek) [reply app/views/profiles/edit.html.erb]: Ok, i will fix it

File: app/controllers/registrations_controller.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class RegistrationsController < Devise::RegistrationsController
+  layout "application", only: [:edit]
+
```
Comment (sniffler) [inline app/controllers/registrations_controller.rb]: Trailing whitespace detected.

File: app/assets/stylesheets/components/devise/_edit_registration.scss [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    color: $gray;
+    display: block;
+    font-size: 14px;
+    margin-bottom: 5px;
+    padding: 0;
+    text-align: left;
+    z-index: 200;
+  }
+
+  >.field,
```
Comment (sniffler) [inline app/assets/stylesheets/components/devise/_edit_registration.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/components/devise/_edit_registration.scss [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    border: 1px solid $signbox-border-color;
+    border-radius: 5px;
+    margin: 0 auto;
+    max-width: 400px;
+    padding: 40px;
+    text-align: left;
+    width: 100%;
+  }
+
+  >.signbox>.button {
```
Comment (sniffler) [inline app/assets/stylesheets/components/devise/_edit_registration.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: app/assets/stylesheets/components/devise/_edit_registration.scss [hunk 5]
```diff
@@ -0,0 +1,70 @@
+.edit-form {
```
Comment (sniffler) [inline app/assets/stylesheets/components/devise/_edit_registration.scss]: Rule set contains (11/10) properties

File: app/assets/stylesheets/components/_main_navigation.scss [hunk 6]
```diff
@@ -3,6 +3,7 @@
   display: flex;
   flex: 0 0 280px;
   flex-direction: column;
+  position: fixed
```
Comment (sniffler) [inline app/assets/stylesheets/components/_main_navigation.scss]: Declaration should be terminated by a semicolon

**Pull request merged**